<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlForm</class>
 <widget class="QWidget" name="ControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wChart" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>550</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>155</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>155</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Генерация случайных
координат</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pbStartRandom">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/Resources.qrc">
              <normaloff>:/gui/img/play.svg</normaloff>:/gui/img/play.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbPauseRandom">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/Resources.qrc">
              <normaloff>:/gui/img/pause.svg</normaloff>:/gui/img/pause.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StopRandom">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/Resources.qrc">
              <normaloff>:/gui/img/stop.svg</normaloff>:/gui/img/stop.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sbX"/>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="sbY"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chbAutoIncrement">
          <property name="text">
           <string>Автоинкремент</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pbAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/Resources.qrc">
              <normaloff>:/gui/img/plus.svg</normaloff>:/gui/img/plus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbDelete">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/Resources.qrc">
              <normaloff>:/gui/img/minus.svg</normaloff>:/gui/img/minus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbEdit">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/Resources.qrc">
              <normaloff>:/gui/img/refresh.svg</normaloff>:/gui/img/refresh.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
